src/tests/account/test_dual_eth_account.cairo

```diff
@@ -211,7 +213,7 @@ fn test_dual_isValidSignature() {
     target.setPublicKey(data.public_key);
 
     let is_valid = camel_dispatcher.is_valid_signature(hash, serialized_signature);
-    assert(is_valid == 'VALID', 'Should accept valid signature');
+    assert_eq!(is_valid, starknet::VALIDATED);
```

Comment:
I think this is much easier to read than jamming the whole `is_valid_signature()` method as an input in the assertion